enterprise/backend/test/metabase_enterprise/cache/task/refresh_cache_configs_test.clj

```diff
@@ -84,9 +84,9 @@
 (deftest schedule-cache-config->card-ids-test
   (mt/with-temp [:model/Dashboard {dashboard-id :id} {:name "Dashboard"}
                  :model/Card {card-id-1 :id} {:name "Cached card 1"
-                                              :dataset_query {:database 1}}
+                                              :dataset_query {:database (mt/id), :type :native, :native {:query "SELECT *"}}}
```

Comment:
Had to fix a handful of tests that created Cards with queries that were never valid, even 5 years ago, now that we end up checking the schemas a little more often